spec/src/main/asciidoc/microprofile-opentracing.asciidoc

```diff
@@ -184,13 +184,13 @@ If there is an exception object available the implementation SHOULD also add log
 [[server-skip-pattern]]
 ===== Disabling server side tracing
 Server side tracing can be disabled by specifying a skip pattern which is used to match with HTTP path
-`UriInfo.getPath()`. If the regex matches with HTTP path then tracing for the given server
-request is disabled even if the method or class is annotated with `@Traced`.
-The configuration does not disable any outbound request made from the disabled
-server endpoint.
+including context path and excluding query parameters e.g. `UriInfo.getAbsolutePath().getPath()`.
+If the regex matches with HTTP path then tracing for the given server request is disabled even if
+the method or class is annotated with `@Traced`. The configuration does not disable any outbound
+request made from the disabled server endpoint.
 
 The skip pattern is specified as a string with key `mp.opentracing.server.skip-pattern` which has to be
-compliant with `java.util.regex.Pattern`. An example skip pattern might be `mp.opentracing.server.skip-pattern=/health|/metrics.*`
+compliant with `java.util.regex.Pattern`. An example skip pattern might be `mp.opentracing.server.skip-pattern/context/=/health|/metrics.*`
```

Comment:
Is this suppose to be `mp.opentracing.server.skip-pattern=/context/health|/metrics.*`.  Actually, the MP endpoints are using "/" context root.  I suggest to change the example to `mp.opentracing.server.skip-pattern=/context/foo|/context/bar.*`

Reply:
good point, fixed